modules/meteroid/proto/api/customers/v1/models.proto

```diff
@@ -49,5 +54,18 @@ message Customer {
   optional google.protobuf.Timestamp archived_at = 10;
   optional google.protobuf.Timestamp created_at = 11;
   optional Address billing_address = 12;
-  optional Address shipping_address = 13;
+  optional ShippingAddress shipping_address = 13;
```

Comment:
please make sure it doesn't generate Option<Option<ShippingAddress>>, same for other fields that have custom types

Reply:
No it looks good, I think we can keep the optional explicit for semantic purpose

```rust
    #[prost(message, optional, tag = "12")]
    pub billing_address: ::core::option::Option<Address>,
    #[prost(message, optional, tag = "13")]
    pub shipping_address: ::core::option::Option<ShippingAddress>,
```